documentation/SA1600.md

```diff
@@ -44,6 +44,36 @@ public string JoinNames(string firstName, string lastName)
 }
 ```
 
+The next example shows a method that is inherited from a base class:
+
+```csharp
+/// <summary>
+/// Example base class.
+/// </summary>
+class Vehicle 
+{
+  /// <summary>
+  /// Accelerates the vehicle.
+  /// </summary>
+  public virtual void accelerate()
```

Comment:
```suggestion
  public virtual void Accelerate()
```

Reply:
Looks good. I could not hide my java background :)